src/SFA.DAS.ApplyService.Web/Extensions/SerializableModelStateDictionaryExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace SFA.DAS.ApplyService.Web.Extensions
+{
+    public static class SerializableModelStateDictionaryExtensions
+    {
+        public static ModelStateDictionary ToModelState(this SerializableModelStateDictionary serializableModelState)
+        {
+            var modelState = new ModelStateDictionary();
+
+            foreach (var data in serializableModelState.Data)
```

Comment:
No null check (not that I'd expect it to be otherwise but it is a public method)

Reply:
Null check on what, sorry?

Reply:
serializableModelState.Data could be null